config/zu.py

```diff
@@ -0,0 +1,56 @@
+"Zulu config with language-specific information."
+
+from pynini import *
+
+GRAPHEMES = union("'", "-",
+                  "A", "B", "C", "D", "E", "F", "G", "H", "I",
+                  "J", "K", "L", "M", "N", "O", "P", "Q", "R",
+                  "S", "T", "U", "V", "W", "X", "Y", "Z",
+                  "a", "b", "c", "d", "e", "f", "g", "h", "i",
+                  "j", "k", "l", "m", "n", "o", "p", "q", "r",
+                  "s", "t", "u", "v", "w", "x", "y", "z")
+
+INITIAL_PUNCTUATION = union('"', "'")
+
+FINAL_PUNCTUATION = union("!", '"', ",", ".", ":", ";", "?")
+
+NUMBERS = union("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")
+
+HYPHEN = acceptor("-")
+
+SPACE = acceptor(" ")
+
+VOWELS = union("A", "E", "I", "O", "U",
+               "a", "e", "i", "o", "u")
+
+SIGMA_STAR = union(*("[{}]".format(i) for i in range(1, 256))
```

Comment:
Should we define SIGMA_STAR in a singular place?  Or would that be more trouble than it's worth?  :)

Reply:
Can be done in a follow up!

Reply:
It's now in the `utils.py` config, and I moved SIGMA_STAR to the actual cdrewrite rule rather than defining it separately.